doc/main.html

```diff
@@ -289,7 +289,13 @@ <h3>Parts</h3>
         <h4>General</h4>
         <p>
           Closely related Engineering Documents may be created as Parts. A Part of a document is a separate Engineering Document and shall be shown to be related to other  
-          by using the same root document number plus a suffix of a hyphen and the Part number to distinguish among Parts (e.g. <code>1020-2:2009</code>, indicating document 1020, Part 2).
+          by using the same root document number plus a suffix of a hyphen and the Part number to distinguish among Parts (e.g. <code>1020-2:2009</code>, indicating document 1020, Part 2). 
+        </p>
+        <p>
+          Part numbers should be continuously ascending from document to document (e.g. <code>1020-1:2009</code>, <code>1020-2:2011</code>, <code>1020-3:2012</code>) unless part numbers are used to organize large sets of multi-part documents.
+        </p>
+        <p>
+          The TC Chair shall assign document numbers in consultation with the document editor. Part numbers may be assigned at the request of the document editor at any time after a Project is approved and shall be assigned before a Final Committee Draft ballot is issued.
```

Comment:
_TC Chair_ is not a defined term, _Technology Committee Chair(s)_ is the term.

I recommend that the Standards Director shall be responsible for assigning the part number in consultation with the Technology Committee Chair(s) since it is the Standards Director that maintains the database of documents.

Reply:
This is not common practice and HO does not track part numbers until documents are published. HO is also not necessarily aware of the part number systems used in some efforts e.g. DC, IMF or SMPTE ST 2110 used to organize documents of a standards suite. HO can't make a judgement on where in such a part number system to place a document because it sometimes requires technical expertise. Additionally, we may not know the history about why certain part numbers are not used. The TC chair and the document editor or DG are better suite to do that. I prefer Steve's proposal.

Reply:
@palemieux @tbause see https://github.com/SMPTE/ag-02/pull/16/commits/b8dc221e0dad64c99dc6a230e03219bedf94fdc5 for update as per discussion with Thomas today. Can wordsmith as needed.